kong/plugins/prometheus/exporter.lua

```diff
@@ -336,6 +368,38 @@ local function metric_data()
     enterprise.metric_data()
   end
 
+  -- Hybrid mode status
+  if cp_metrics then
+    -- Cleanup old metrics
+    metrics.data_plane_last_seen:reset()
+    metrics.data_plane_config_hash:reset()
+
+    for data_plane, err in kong.db.clustering_data_planes:each() do
+      if err then
+        kong.log.err("failed to list data planes: ", err)
+        goto next_data_plane
+      end
+
+      local labels = { data_plane.id, data_plane.hostname, data_plane.ip }
+
+      metrics.data_plane_last_seen:set(data_plane.last_seen, labels)
+      metrics.data_plane_config_hash:set(config_hash_to_number(data_plane.config_hash), labels)
```

Comment:
What is the utility of this number?
It can help detect if a data plane is stuck at a *different* version than others but this is not a linearly increasing number so the absolute number is of zero help.

@wyndigo Any thoughts?

Reply:
Yes this is to detect inconsistent config between DPs, ideally they should compare to the hash on current CP,
but due to the API change I'm currently not able to just let it export. Need some change on Kong side.
I will address that in later PRs.
But for now, inconsistency of config hashes through DP already indicates something bad. One can monitor those
with `count_values(data_plane_config_hash) > 1`